Changes

```diff
@@ -45,6 +45,9 @@ Working version
 
 ### Standard library:
 
+- #9248: Add Printexc.default_uncaught_exception_handler
+  (Raphael Sousa Santos)
```

Comment:
could you add `review by Daniel Bünzli`?

Reply:
Done.